docs/overview/index.md

```diff
@@ -135,11 +135,18 @@ There are also limitations on the number of sampling points for which you reques
 
 The Mapzen Elevation lookup service is built from the [Valhalla](https://github.com/valhalla) open-source project.
 
+#### Who's On First API
+
+[Who's On First's API](https://mapzen.com/documentation/wof/) provides all sorts of data about places from our gazetteer 
```

Comment:
same comment as above with our, remove the possessive, and add a period at the end.

The Who's on First API...from the global gazetteer.